content/documentation/pages/1-installation/1-local/3-manual.md

```diff
@@ -0,0 +1,104 @@
+---
+path: 'installation/local/manual'
+title: 'Manual'
+description: 'Manual installation'
+---
+
+# Manual Installation
+
+If Docker does not suit your needs, you can manually install the parts you need to run Spring Cloud Data Flow.
+
+## Download server jars
+
+1.  Download the Spring Cloud Data Flow Server by using the following
+    command:
+
+        wget https://repo.spring.io/{version-type-lowercase}/org/springframework/cloud/spring-cloud-dataflow-server/{project-version}/spring-cloud-dataflow-server-{project-version}.jar
+
+2.  Download [Skipper](https://cloud.spring.io/spring-cloud-skipper/) (because
+    Data Flow delegates to Skipper for those features), by running the
+    following commands:
+
+        wget https://repo.spring.io/{skipper-version-type-lowercase}/org/springframework/cloud/spring-cloud-skipper-server/{skipper-version}/spring-cloud-skipper-server-{skipper-version}.jar
+
+        wget https://repo.spring.io/{skipper-version-type-lowercase}/org/springframework/cloud/spring-cloud-skipper-shell/{skipper-version}/spring-cloud-skipper-shell-{skipper-version}.jar
+
+## Install messaging middleware
+
+These instructions require that RabbitMQ be running on the same machine as Skipper and the Spring Cloud Data Flow server and shell.
```

Comment:
We defaulted to kafka in the docker compose.   Should we stick with Kafka?

Reply:
There is a TODO to address this.